src/modrover/synthesizer.py

```diff
@@ -1,65 +1,70 @@
 from typing import List
 
 import numpy as np
-import pandas as pd
 from numpy.typing import ArrayLike
 
-from .info import SynthSpecs
-
 
 def metrics_to_weights(metrics: ArrayLike,
```

Comment:
Could be worth a discussion where this method belongs. I didn't move it but maybe we should have an `ensemble` submodule or something like that